packages/design-tokens/docs/components/TokensTable.tsx

```diff
@@ -18,15 +18,17 @@ import { TokenName } from "./TokenName";
 
 export const TokensTable = ({
   data,
+  name,
```

Comment:
I noticed that we are using `category` in the type. What do you think of also using it here? Or maybe changing there to `name` so we have the same naming in both places?

Reply:
good call! I'm using category